src/extract-kindle-book.ts

```diff
@@ -44,11 +46,23 @@ async function main() {
   const krRendererMainImageSelector = '#kr-renderer .kg-full-page-img img'
   const bookReaderUrl = `https://read.amazon.com/?asin=${asin}`
 
+    //Switch for multi-OS operation
+    const getChromeExecutablePath = () => {
+      switch (os.platform()) {
+        case 'win32':
+          return 'C:/Program Files/Google/Chrome/Application/chrome.exe';
+        case 'darwin':
+          return '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
+        default:
+          return '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
+      }
+    }
+    const chromePath = getChromeExecutablePath();
```

Comment:
wouldnt merge this in this PR as it's unrelated, and hardcoding paths is not guaranteed, e.g. i do not use chrome and use a custom build of chromium